app/src/main/java/org/sil/storyproducer/model/Phase.kt

```diff
@@ -3,146 +3,176 @@ package org.sil.storyproducer.model
-            PhaseType.CONSULTANT_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.DRAMATIZATION -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.BACKT -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.TRANSLATE_REVISE -> Workspace.activeStory.slides[slideNum].narrationFile
+            PhaseType.COMMUNITY_WORK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.ACCURACY_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.VOICE_STUDIO -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.BACK_T -> Workspace.activeStory.slides[slideNum].chosenDraftFile
             else -> ""
         }
         return Story.getFilename(filename)
     }
 
+    /**
+     * TODO: refactor
+     * the following 3 functions are used when displaying the phase names and creating paths to store audio files
+     * (see Issue #547 for conversation on fix)
+     */
     fun getPrettyName() : String {
         return when (phaseType) {
             PhaseType.LEARN -> "Learn"
-            PhaseType.DRAFT -> "Translate"
-            PhaseType.CREATE -> "Finalize"
+            PhaseType.TRANSLATE_REVISE -> "Translate + Revise"
+            PhaseType.FINALIZE -> "Finalize"
             PhaseType.SHARE -> "Share"
-            PhaseType.COMMUNITY_CHECK -> "Community Work"
-            PhaseType.CONSULTANT_CHECK -> "Accuracy Check"
+            PhaseType.COMMUNITY_WORK -> "Community Work"
+            PhaseType.ACCURACY_CHECK -> "Accuracy Check"
             PhaseType.WHOLE_STORY -> "Whole Story"
             PhaseType.REMOTE_CHECK -> "Remote Check"
-            PhaseType.BACKT -> "Back Translation"
-            PhaseType.DRAMATIZATION -> "Voice Studio"
+            PhaseType.BACK_T -> "Back Translation"
+            PhaseType.VOICE_STUDIO -> "Voice Studio"
             else -> phaseType.toString().toLowerCase()
         }
     }
-
     fun getDisplayName() : String {
         return when (phaseType) {
-            PhaseType.DRAFT -> "Translation Draft"
-            PhaseType.COMMUNITY_CHECK -> "Comment"
-            PhaseType.CONSULTANT_CHECK -> "Accuracy"
+            PhaseType.TRANSLATE_REVISE -> "Translation Draft"
+            PhaseType.COMMUNITY_WORK -> "Comment"
+            PhaseType.ACCURACY_CHECK -> "Accuracy"
             PhaseType.WHOLE_STORY -> "Whole"
             PhaseType.REMOTE_CHECK -> "Remote"
-            PhaseType.BACKT -> "BackTrans"
-            PhaseType.DRAMATIZATION -> "Studio Recording"
-            PhaseType.CREATE -> "Finalize"
+            PhaseType.BACK_T -> "BackTrans"
+            PhaseType.VOICE_STUDIO -> "Studio Recording"
+            PhaseType.FINALIZE -> "Finalize"
             else -> phaseType.toString().toLowerCase()
         }
     }
-
     fun getShortName() : String {

```

Comment:
I think you had a clearer function name something like getFileSafeName() which I thought was an improvement that you could retain.

Reply:
I'll add these back.